contrib/epee/include/net/http_base.h

```diff
@@ -136,22 +110,22 @@ namespace net_utils
 			}
 		};
 
-    struct uri_content
-    {
-      std::string m_path;
-      std::string m_query;
-      std::string m_fragment;
-      std::list<std::pair<std::string, std::string> > m_query_params;
-    };
+	struct uri_content
```

Comment:
Revert whitespace only changes in this file :/ kind of a rule around here.

Reply:
Done.